### PR TITLE
Club Styling

### PR DESCRIPTION
**Pull request by steven-solar**
Add section headers, spacing. 

Screenshot 2020-08-05 at 1 13 18 PM

Screenshot 2020-08-05 at 1 13 22 PM

Screenshot 2020-08-05 at 1 13 26 PM
Screenshot 2020-08-05 at 1 13 32 PM

### CODE REVIEW

**Review by MauroGuerrero20 — APPROVED** [review]
LGTM

**Review by antoniolinhart — APPROVED** [review]
Looks good! I think this will be much cleaner for the users to see on their page with the subtitles

File: capstone-frontend/src/components/ClubPage.jsx [hunk 1]
```diff
@@ -257,41 +266,41 @@ class ClubPage extends Component {
               <hr className='light-gray-border mx-2 my-2' />
 
               <div className='text-center'>
+                <h3 className='mt-3'> Club Descrpition </h3>
```
Comment (VivekShankar) [inline capstone-frontend/src/components/ClubPage.jsx]: nit: Description

File: capstone-frontend/src/components/AdminClubPage.jsx [hunk 2]
```diff
@@ -20,23 +20,24 @@ class AdminClubPage extends Component {
   fetchData = () => {
     let requesters = [];
     let members = [];
-    fetch(`/api/clubs?id=${this.props.id}`)
+    let club; 
+    fetch(`/api/clubs?id=${this.props.match.params.id}`)
       .then(response => response.json()).then(clubs => {
-        const club = clubs[0];
-        this.setState({ club, requesters: [], members: [] });
+        club = clubs[0];
```
Comment (VivekShankar) [inline capstone-frontend/src/components/AdminClubPage.jsx]: nit: Can this remain a `const`?

**Review by VivekShankar — APPROVED** [review]
LGTM

File: capstone-frontend/src/components/ClubPage.jsx (hunk 1 above)
Comment (steven-solar) [reply capstone-frontend/src/components/ClubPage.jsx]: Will fix!

File: capstone-frontend/src/components/AdminClubPage.jsx (hunk 2 above)
Comment (steven-solar) [reply capstone-frontend/src/components/AdminClubPage.jsx]: Sure! Was changing things around to test, and forgot to revert this to a const.

**Pull request merged**